.github/workflows/build.yml

```diff
@@ -14,11 +14,23 @@ jobs:
       - name: Read .nvmrc
         run: echo "##[set-output name=NVMRC;]$(cat .nvmrc)"
         id: nvm
-      - uses: actions/setup-java@v1
+      - uses: actions/setup-java@v2
         with:
           java-version: 8
+          distribution: 'adopt'
+          server-id: ossrh
+          server-username: MAVEN_USERNAME
+          server-password: MAVEN_CENTRAL_TOKEN
+          gpg-private-key: ${{ secrets.SIGNING_KEY }}
+          gpg-passphrase: MAVEN_GPG_PASSPHRASE
       - uses: actions/setup-node@v2
         with:
           node-version: '${{ steps.nvm.outputs.NVMRC }}'
       - run: |
           ./mvnw -V -B verify
+          npm ci
+          npx semantic-release
+        env:
+          MAVEN_USERNAME: eller86
```

Comment:
Is there a reason to do this instead of just directly using it on line 24?

Reply:
Because [`server-username`](https://github.com/actions/setup-java/blob/0443912ed73df2ee8596160daba017daab1c8ca9/action.yml#L32-L34) needs to be name of an environment variable that specify actual Maven username. So on line 24, I set `MAVEN_USERNAME` (that is `GITHUB_ACTOR` by default) and set the environment variable at here.

You may refer the [official guideline to configure actions/setup-java for Maven and GitHub Actions](https://github.com/actions/setup-java/blob/0443912ed73df2ee8596160daba017daab1c8ca9/docs/advanced-usage.md#Publishing-using-Apache-Maven) for detail. :)